File: 백준/Bronze/30468. 호반우가 학교에 지각한 이유 1/호반우가 학교에 지각한 이유 1.java
import java.io.*;
import java.lang.*;
import java.util.*;

public class Main {

    private static int N;
    private static int STR, DEX, INT, LUK;

    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));

        input(br);

        int nn = N * 4;
        int sum = STR + DEX + INT + LUK;

        bw.write(String.valueOf(Math.max(0, nn - sum)) + '\n');

        br.close();
        bw.close();
    }

    private static void input(BufferedReader br) throws IOException {
        StringTokenizer st = new StringTokenizer(br.readLine());

        STR = Integer.parseInt(st.nextToken());
        DEX = Integer.parseInt(st.nextToken());
        INT = Integer.parseInt(st.nextToken());
        LUK = Integer.parseInt(st.nextToken());

        N = Integer.parseInt(st.nextToken());
    }
}

File: 백준/Bronze/30468. 호반우가 학교에 지각한 이유 1/호반우가 학교에 지각한 이유 1.py
A, B, C, D, N = map(int, input().split())
E = (A+B+C+D) / 4
F = (A+B+C+D) // 4

if E >= N:
    print(0)
else:
    H = A+B+C+D
    if H%4 == 0:
        print(4*(N-F))
    else:
        print(4*N-H)